payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/KlarnaCountrySpec.kt

```diff
@@ -20,13 +20,12 @@ data class KlarnaCountrySpec(
     override val apiPath: IdentifierSpec = IdentifierSpec.Country
 ) : FormItemSpec() {
     fun transform(
-        currencyCode: String?,
         initialValues: Map<IdentifierSpec, String?>
     ) = createSectionElement(
         CountryElement(
             this.apiPath,
             DropdownFieldController(
-                CountryConfig(KlarnaHelper.getAllowedCountriesForCurrency(currencyCode)),
+                CountryConfig(),
```

Comment:
I'm not familiar with how Android works, but do we need this entire class if it is just now a normal country list?

Reply:
Hmm, I can try to replace it with our normal country placeholder. But we can't rip out `klarna_country` from most of the things, just map it to our default CountrySpec.